src/shims/mod.rs

```diff
@@ -65,4 +46,39 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         // Otherwise, load the MIR.
         Ok(Some(this.load_mir(instance.def, None)?))
     }
+
+    fn align_offset(
+        &mut self,
+        ptr_op: OpTy<'tcx, Tag>,
+        align_op: OpTy<'tcx, Tag>,
+        layout: ty::layout::TyLayout<'tcx>,
+    ) -> InterpResult<'tcx, u128> {
+        let this = self.eval_context_mut();
+
+        let req_align = this.force_bits(
+            this.read_scalar(align_op)?.not_undef()?,
+            this.pointer_size()
+        )? as usize;
+
+        // FIXME: This should actually panic
```

Comment:
```suggestion
        // FIXME: This should actually panic in the interpreted program
```
I was confused for a sec.^^ Maybe just because I am tired but whatever.